frontend/src/features/public-form/PublicFormProvider.tsx

```diff
@@ -260,7 +267,13 @@ export const PublicFormProvider = ({
               // Using catch since we are using mutateAsync and react-hook-form will continue bubbling this up.
               .catch((error) => {
                 // TODO: Remove when we have resolved the Network Error and t.arrayBuffer issues.
-                console.warn(formInputs)
+                datadogLogs.logger.warn('handleSubmitForm', {
+                  meta: {
+                    action: 'handleSubmitForm',
+                    formInputs: formInputs,
```

Comment:
I'm a little worried about PII or sensitive information leakage. Let's try to mask whatever sensitive information we can identify like phone number and emails.

Reply:
[abd41f0](https://github.com/opengovsg/FormSG/pull/5667/commits/abd41f0a6f810a5bf8644804ac906d496d2a9db7)